### ISSUE TITLE
Can not connect to snowflake DB

### ISSUE THREAD

**Issue by jonhilgart22** [TablePlus/TablePlus#1681]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Snowflake

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
286

3. The steps to reproduce this issue:
I can't connect to a snowflake DB

Noted: If the bug is related to data, please attach an example SQL data.
![image](https://user-images.githubusercontent.com/8621935/73113033-62304a00-3ec6-11ea-9d6b-9b705f020ff4.png)

**Comment by jonhilgart22** [comment]
Is there some guidance on how to troubleshoot connecting to a snowflake DB? The error messages are not super helpful.

**Comment by huyphams** [comment]
Hi @jonhilgart22 please try to use the account with this format:
<img width="268" alt="Screen Shot 2020-01-25 at 7 26 47 AM" src="https://user-images.githubusercontent.com/3654622/73120425-2045e900-3f44-11ea-8609-773c45498aa8.png">

**Comment by jonhilgart22** [comment]
That still doesn't work.

![image](https://user-images.githubusercontent.com/8621935/73203211-76a45a80-40f1-11ea-8e68-3d19c2a2e2eb.png)

**Comment by jonhilgart22** [comment]
Is it possible to get more verbose error logging? Any other things to try? 

I appreciate the help...I'd hate to have to switch away from TablePlus because I can't connect to Snowflake.

**Comment by huyphams** [comment]
Hi @jonhilgart22 did you use the correct account? here is mine:
<img width="612" alt="Screen Shot 2020-01-28 at 8 02 04 AM" src="https://user-images.githubusercontent.com/3654622/73226982-8761d880-41a4-11ea-8d93-5cefea794585.png">

**Comment by jonhilgart22** [comment]
yes - I'm able to log in via the web client but not this client. Can I enable more verbose error messages?

Colleagues have been able to connect via other SQL clients.

**Comment by flogiston** [comment]
Same here, no success connecting (with all account ID, username/password connections) and zero debugging information on either TablePlus or Snowflake side. SSH tunnel works fine, only the connection to Snowflake fails. Any hints, anyone? Thanks!

**Comment by JohnRomanski** [comment]
this issue could be solved or considered in looking at https://github.com/TablePlus/TablePlus/issues/1150

**Comment by al1p-R** [comment]
Same issue with TablePlus Version 4.5.2 (403)

not using authenticator - #1150 did not help.

Quite uninformative error message:

![20220127_ TablePlus_000116](https://user-images.githubusercontent.com/34063534/151368461-fc751c0d-c7ac-4f64-bf14-56186097a0fd.png)

I can connect to Snowflake using these credentials with DataGrip or SQLPro Studio.

Any hint on how to make Snowflake work with TablePlus ?

**Comment by aditya-pattern** [comment]
@al1p-R 

for DataGrip and SQLPro Studio use full account name
xxx.snowflakecomputing.com - for west
xxx.region.snowflakecomputing.com - for others

for TablePlus use
xxx - for west
xxx.region - for others

For other params the below worked for me
Authenticator - snowflake
Port - leave empty
and password saved in keychain option

**Comment by Chekaloff** [comment]
Why not save your users some time with simple screenshots TablePlus?

This options for authenticator:

<img width="1676" alt="CleanShot 2023-01-25 at 22 55 44@2x" src="https://user-images.githubusercontent.com/113585648/214755589-c4ed9834-880c-4595-b230-4751eda1f7c9.png">


then 

for my demo account 

https://vm98735.ca-central-1.aws.snowflakecomputing.com .... 


Account: vm98735.ca-central-1.aws
Authenticator: SNOWFLAKE


<img width="535" alt="CleanShot 2023-01-25 at 22 59 02@2x" src="https://user-images.githubusercontent.com/113585648/214755894-bdee8ea1-9e3b-435c-bc27-dafe5cba52c3.png">


<img width="535" alt="CleanShot 2023-01-25 at 22 59 47@2x" src="https://user-images.githubusercontent.com/113585648/214755901-bc894192-ba2d-4b48-a542-1539d4f3db1e.png">

**Comment by joe-rh** [comment]
I would have never been able to connect to Snowflake with TablePlus without this post.

**Comment by avivzk** [comment]
Thanks! Went through 3 weeks of trying everything until I found this issue

**Comment by nickolasclarke** [comment]
Using the URIs in various forms did not work for me. I tried `M****-*****.snowflakecomputing.com` using the account ID as the subdomain, as well as `B******.AWS_US_EAST_1.AWS` both upper and lower case, and replacing `_` for `-` and a few other permutations as well. 

I had to use the uppercase account ID alone, and explicitly set `Authenticator` to `SNOWFLAKE`. The fact that there is no error message at all besides `error` is quite frustrating. Can you please expose more information? 

![Image](https://github.com/user-attachments/assets/096612c4-b0ef-4699-a2db-fdd08bdf4c24)
